tests_metricflow/integration/query_output/test_query_output.py

```diff
@@ -291,3 +291,30 @@ def test_saved_query_override_order_by_and_limit(  # noqa: D103
         snapshot_str=query_result.result_df.text_format(),
         sql_engine=sql_client.sql_engine_type,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+@pytest.mark.duckdb_only
+def test_semi_additive_measure_with_where_filter(  # noqa: D103
```

Comment:
We previously only had a check query test for semi additive joins with a where filter, so this adds a more explicit check.